Modules/_functoolsmodule.c

```diff
@@ -70,23 +145,30 @@ get_functools_state_by_type(PyTypeObject *type)
 static PyObject *
 partial_new(PyTypeObject *type, PyObject *args, PyObject *kw)
 {
-    PyObject *func, *pargs, *nargs, *pkw;
+    PyObject *func, *pto_args, *new_args, *pto_kw;
```

Comment:
Why are these variables renamed?

Reply:
I did some renaming to make variable names unambiguous and consistent across different methods.

https://github.com/python/cpython/pull/119827#issuecomment-2180683652

Would this be useful to add to comments?

Reply:
Don't worry about it. It makes reviewing and future backportings more difficult, but if it makes the code better in long term, it is okay.